libs/net/dal/Services/MediaTypeService.cs

```diff
@@ -20,5 +24,53 @@ public IEnumerable<MediaType> FindAll()
     {
         return this.Context.MediaTypes.OrderBy(a => a.SortOrder).ThenBy(a => a.Name).ToArray();
     }
+
+    public IPaged<MediaType> Find(MediaTypeFilter filter)
+    {
+        var query = this.Context.MediaTypes
+            .Include(c => c.Name)
+            .Include(c => c.Description)
+            .Include(c => c.IsEnabled)
+            .AsQueryable();
+
+        if (!String.IsNullOrWhiteSpace(filter.Name))
+            query = query.Where(c => EF.Functions.Like(c.Name, $"%{filter.Name}%"));
+        if (!String.IsNullOrWhiteSpace(filter.Description))
+            query = query.Where(c => c.Description == filter.Description);
+
+        var total = query.Count();
+
+        if (filter.Sort?.Any() == true)
+            query = query.OrderByProperty(filter.Sort);
```

Comment:
Need to fix this with the current code we're using within the `ContentService.Find` method.  I just fixed this bug.